_posts/2016-05-18-mid-may-features-update.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Mid-May Features Update"
+author: "Hayley MacLeod"
+categories: pitchplay
+---
+
+## Follow and Followers
+
+Do your friends practice? Motivate and prepare to be motivated by following them on PitchPlay. Go to your profile, copy your URL and send it to friends.
+
+To get to your profile, hover over your profile image on the far right of your screen and click __Profile__.
+
+![](/assets/img/2016-05-13/gotoprofile.png){:width="25%"}
+
+Copy your URL. Chances are it looks something like mine, only with your name:
+
+__[https://pitchplay.io/user/hayley-macleod](https://pitchplay.io/user/hayley-macleod)__
+
+Send your URL link to friends so that they can follow you. Once you receive a friend's URL link, you can go to their profile and follow them. Following a friend will enable their activity on PitchPlay to show up in your activity feed. If you are being followed by friends, your activity will show up in their activity feeds.
+
+## Activity Feed
+
+The activity feed is one way you can stay up-to-date on what has been going on in PitchPlay's social world. You'll see when friends practice, follow and are followed. Additionally, you'll see your own practice sessions populating within your feed.
+
+![](/assets/img/2016-05-13/activityfeed.png){:width="50%"}
+
+---
+
+## <center> New Tools in Your Practice Canvas
+
+### Mirror
+
+Have you ever watched a video of yourself playing your instrument and recognized a bad habit?
```

Comment:
I think this would work better as a descriptive statement instead of an inquisitive one. e.g. `Practicing in front of a mirror is a great way to recognize bad habits`. We're not actually calling anything to action here, it's not really possible for the user to reply to the question (there is the comments section below, but you'd want any kind of CTA to be near the end so it doesn't break flow of the rest of the article)